packages/controls/src/breakpoints/breakpoints.ts

```diff
@@ -161,3 +175,48 @@ export function mergeResponsiveValues<A>(
     },
   })) as DeviceOverride<A>[]
 }
+
+export function findNextFallback<V>(
+  breakpoints: Breakpoints,
+  value: ResponsiveValue<V>,
+  deviceId: Breakpoint['id'],
+  activeDeviceId: Breakpoint['id'],
+  fallbackStrategy?: FallbackStrategy<V>,
+): Breakpoint | null {
```

Comment:
Moved from https://github.com/makeswift/makeswift/pull/947/files#diff-98cfbae694c1ec54b413804d485fd52c751a1e706e56312369ea0b9e7f336e2dL184, adjusted the `return` statement logic to reflect that reflect that `getBreakpoint` throws ([from day one](https://github.com/makeswift/makeswift/commit/8303ed66653e66e8a97a10cc04adccc35a610a6f#diff-98cfbae694c1ec54b413804d485fd52c751a1e706e56312369ea0b9e7f336e2dR63)) rather than returning `null` / `undefined` when the device ID is not found in the breakpoints.